_pages/en/strategy-build-vs-buy.md

```diff
@@ -118,12 +134,13 @@ Stakeholders are expected to still comply with existing policy instruments inclu
 ### Legal
 
 1. IT Solutions use the ESDC Software Legal Terms Guidance (to be created - see [Coherent set of actions](#coherent-set-of-actions))
-2. IT Solutions and IT Products comply with the ESDC OSS Management Framework
+2. IT Solutions and IT Products comply with the ESDC Open Source Software Management Framework when applicable.
 
 ### Architecture
 
-1. IT Solutions and IT products loose coupling boundaries are at the Business Capability Model level.
-   - Duplication of IT Products or IT Solutions is supported as long as they are for different business capabilities.
+1. IT Solutions and IT products loose coupling boundaries are at intersection of value stream, value stage and business capability (ESDC Business Capability Model).
+   - Duplication of IT Solutions or IT Products is supported as long as they are for different business capabilities.
+   - Boundary of IT Solutions and IT Products are to be application capabilities so that they can enable business capabilities.
 2. IT Products are risk assessed for lock-in situations with the IT Products risk assessment framework (to be created - see [Coherent set of actions](#coherent-set-of-actions))
 
 ## Coherent set of actions
```

Comment:
Question
At Governance / Produce the EARB standard definition of IT Solutions. Do we mean to formally define the relationships between IT Solutions, IT Products, Software, Infrastructure?

i.e. What is in Appendix A - Business Case / Build vs Buy : Oversimplified Question

Reply:
I think we might need to do some level of explanations? The TOGAF model is great when you look at the acquisition of IT Solutions (bunch of key concepts to look into) but covers barely anything for the "operations" of that solution. That's mostly covered under ITIL instead where we define the various processes required to ensure SLA are met.

Another dimension we're trying to bring in this version is the breakdown of the rights and conditions we acquire an IT Solution (whether adopted, bought or built) and the actual maintenance (software patches, security fixes, feature enhancements, etc.) of it.

Reply:
Probably need to create issue for this.

Reply:
created issue #1332